### ISSUE TITLE
/scripts/test.sh used by yarn test and referenced in CONTRIBUTING.md does not exist

### ISSUE THREAD

**Issue by jelhan** [ef4/ember-auto-import#346]
A file `/scripts/test.sh` does not exist. It also does not seem to be created by `yarn compile`.

But it is used in `yarn test`. https://github.com/ef4/ember-auto-import/blob/c799048a5cc969a7c4ec7ca9d72f677dd16cc119/package.json#L5

And it is referenced in CONTRIBUTING.md:

> The top-level test runner script (`/scripts/test.sh`) probably only works under Unix-like environments, because it relies on GNU parallel to run all the apps at once.

It was deleted in #333.

To be honest I wasn't sure how to run all tests locally without reproducing the tests in GitHub Actions workflow.

**Comment by ef4** [comment]
Yeah, sorry, this is annoying at the moment. I'm in the middle of refactoring how the tests work, see [branch layered-tests](https://github.com/ef4/ember-auto-import/tree/layered-tests).

**Comment by jelhan** [comment]
Not sure what goals you are trying to achieve with the refactoring. But just in case you haven't seen it yet: [ember-addon-tests](https://github.com/jelhan/ember-addon-tests). Basically as the good old [ember-cli-addon-tests](https://github.com/tomdale/ember-cli-addon-tests) from Tom Dale but using a yarn monorepository under the hood. Also modernized API.

It is still very alpha. I build it to improve the tests for [ember-cli-content-security-policy](https://github.com/rwjblue/ember-cli-content-security-policy). And it works very well for that use case so far. It might be beneficially for ember-auto-import as well. At least the use case seems to be similar.
